docs/pages/project/contributing/contributing-docs.md

```diff
@@ -298,6 +298,48 @@ The output of the code snippet would be:
       Thing 3
 ```
 
+In this example, **Group 1** is a ``parent`` section within the Table of Contents, and **Thing 1**, **Thing 2**, and **Thing 3** are ``children`` of **Group 1**. This is a direct hierarchy with a single level of children under the parent.
+
+``Grandchildren`` are added in the sidebars in the following manner:
+
+```
+    toc:
+  - title: Group 1
+    subfolderitems:
+      - page: Thing 1
+        url: /thing1.html
+        subfolderitems:
+          - page: Subthing 1.1
+            url: /subthing1_1.html
+          - page: Subthing 1.2
+            url: /subthing1_2.html
+      - page: Thing 2
+        url: /thing2.html
+      - page: Thing 3
+        url: /thing3.html
+```
+
+The output of the code snippet would be:
+
+```
+    Group 1
+      Thing 1
+       Subthing 1.1
+       Subthing 1.2
+      Thing 2
+      Thing 3
+```
+
+Here, **Subthing 1.1** and **Subthing 1.2** are the ``grandchildren`` of **Thing 1**.
+
+In `docs/_includes/sidebar.html` contains three levels of navigation hierarchy.
+
+- **Parent**: It serves as a top level category for related content.
```

Comment:
Parent isn't explicitly a thing, so please ensure that you are listing implicitly.